### PR TITLE
(PC-7175) App native - Favoris - Ecran si pas connecté

### PR DESCRIPTION
**Pull request by kopax-polyconseil**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-7175

## Checklist

I have:

- [x] Made sure the title of my PR follows the convention `[PC-XXX] <summary>`.
- [x] Made sure my feature is working on the relevant real / virtual devices.
- [x] Written **unit tests** for my feature.
- [x] Added new reusable components to the AppComponents page and communicate to the team on slack
- [x] Added a **screenshot** for UI tickets.
- [ ] Attached a **ticket number** for any added TODO/FIXME \
       (for tech tasks, give the best context about the TODO resolution: what/who/when).

## Screenshots

| \*iOS - [Phone name] | \*Android - [Phone name] | Tablet - [Phone name] | Little - [Phone name] |
| -------------------- | :----------------------: | --------------------: | --------------------: |
|                      |image|                       |                       |

## Deploy hard

If native code (ios/android) was modified, **after** the PR is merged, on the master branch, upgrade the **app version** (+1 patch):

- if you want an hard deployment of the testing environment, use `yarn trigger:testing:deploy`

### CODE REVIEW

File: src/ui/svg/icons/CreateAccount.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+import { IconInterface } from './types'
+
+export const CreateAccount: React.FunctionComponent<IconInterface> = ({
+  size = 127,
+  color = ColorsEnum.BLACK,
+  testID,
+}) => (
+  <Svg
+    width={Math.round(1.55 * (size as number))}
```
Comment (BenStirrup) [inline src/ui/svg/icons/CreateAccount.tsx:14]: 1. Il sort d'ou ce calcul ?
2. 
```suggestion
    width={Math.round(1.55 * size))}
```

File: src/features/favorites/pages/__tests__/Favorites.test.tsx [hunk 2]
```diff
@@ -29,6 +29,11 @@ describe('Favorites component', () => {
     expect(toJSON()).toMatchSnapshot()
   })
 
+  it('should show non connected page when not logged in', () => {
+    const { getByText } = renderFavorites(false)
+    expect(getByText('Connecte-toi pour profiter de cette fonctionnalité !')).toBeTruthy()
+  })
+
```
Comment (BenStirrup) [inline src/features/favorites/pages/__tests__/Favorites.test.tsx:36]: 1. `const mockUseAuthContext = useAuthContext as jest.MockedFunction<...>` au fait
2. `renderFavorites({ isLogged: false })` ==> les named arguments manquent vraiment à JavaScript

File: src/features/favorites/pages/Favorites.tsx [hunk 3]
```diff
@@ -36,7 +36,7 @@ const useShowResults = () => {
 export const FavoritesScreen: React.FC = () => {
   useKeyboardAdjust()
   const { dispatch } = useFavoritesState()
-  const showResults = useShowResults()
+  useShowResults()
```
Comment (BenStirrup) [inline src/features/favorites/pages/Favorites.tsx:39]: C'est normal que ce custom hook ne renvoie rien ?

File: src/ui/svg/icons/CreateAccount.tsx (hunk 1 above)
Comment (kopax-polyconseil) [reply src/ui/svg/icons/CreateAccount.tsx:14]: C'est un ratio, sans le cast il y a un warning

File: src/features/favorites/pages/__tests__/Favorites.test.tsx (hunk 2 above)
Comment (kopax-polyconseil) [reply src/features/favorites/pages/__tests__/Favorites.test.tsx]: OK

File: src/features/favorites/pages/Favorites.tsx (hunk 3 above)
Comment (BenStirrup) [reply src/features/favorites/pages/Favorites.tsx]: J'espère que c'est testé alors
Un jour je le supprimerai et on verra si un test casse^^

Comment (kopax-polyconseil) [reply src/features/favorites/pages/Favorites.tsx]: J'ai rename & clean le hook pour que ça reflète ce qui est nécessaire pour le ticket.

File: src/ui/svg/icons/CreateAccount.tsx (hunk 1 above)
Comment (kopax-polyconseil) [reply src/ui/svg/icons/CreateAccount.tsx:14]: Le calcul du ratio c'est `viewBox.width / viewBox.height`

**Pull request merged**